### PR TITLE
1.1.0-rc1 Updates

### LINKED ISSUE #246: 1.1.0-RC1 Issue - launch wizard
Issues inside of the rc1 release of 1.1.0:

- launch wizard is broken and non functional inside of the VSIX for rc1 release of 1.1.0

### LINKED ISSUE #247: 1.1.0-RC1 Issue - Language file not added
- Notices in the console a language file was not added to the VSIX but is needed. this file is `src/language/providers/intellisense/DFDLGeneralFormat.dfdl.xsdf`
  - Might have caused #246 

Other minor item that will be part of this PR:
- Minor thing the root `.vscodeignore` needs removed as it was moved inside of `build/package`

### PR DESCRIPTION
**Pull request by Shanedell**
1.1.0-rc1 Updates:

- Add needed language file to extension package
  - This being missing is what caused the launch wizard along with other things to break
- Remove the root .vscodeignore as it is no longer needed

Closes #246
Closes #247

### CODE REVIEW

**Review by scholarsmate — APPROVED** [review]

File: build/package/.vscodeignore [hunk 1]
```diff
@@ -29,3 +29,4 @@
 !src/omega_edit/omega_edit.js
 !src/omega_edit/interface.html
 !src/omega_edit/omega-edit-scala-server*.zip
+!src/language/providers/intellisense/DFDLGeneralFormat.dfdl.xsdf
```
Comment (stevedlawrence) [inline build/package/.vscodeignore:32]: Is the `f` at the end of `.dfdl.xsdf` intentional? The file included with Daffodil just has ` .dfdl.xsd` extension. Is this so it doesn't conflict with what Daffodil distributes or something, or is this just a typo? It looks like the actual file does have the `f` as well as a path in `src/language/dfdl.ts`, so if this is typo we are consistent.

Comment (Shanedell) [reply build/package/.vscodeignore:32]: Yeah it is a typo, but a consistent typo will fix that

**Review by stevedlawrence — APPROVED** [review]
+1

**Comment by Shanedell** [comment]
@tuxji @mbeckerle @arosien Would one of you guys be able to review this PR when you have a minute?

**Review by tuxji — APPROVED** [review]
+1

**Review by scholarsmate — APPROVED** [review]

**Pull request merged**